src/js_results.hpp

```diff
@@ -49,6 +50,7 @@ class Results : public realm::Results {
 
 template<typename T>
 struct ResultsClass : ClassDefinition<T, realm::js::Results<T>, CollectionClass<T>> {
+    using Type = T;
```

Comment:
seems not used

Reply:
It's used inside `compute_aggregate_on_collection`: `get_internal<typename T::Type, T>(this_object)`